testutil/network/network.go

```diff
@@ -121,6 +121,8 @@ func newGenAccout(kr keyring.Keyring, name string, amount int64) (authtypes.Gene
 	balances := sdk.NewCoins(
 		sdk.NewCoin(fmt.Sprintf("%stoken", name), sdk.NewInt(amount)),
 		sdk.NewCoin(app.BondDenom, sdk.NewInt(amount)),
+		sdk.NewCoin(app.DisplayDenom, sdk.NewInt(amount)),
```

Comment:
we have the correct token denomination, so we don't have to hold up this PR, but we might also want to get rid of giving CELES tokens just to avoid any confusion for future readers.

approving without tho